### PR TITLE
Heroku setup

### PR DESCRIPTION
**Pull request by sbrych**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "webpack": "^1.12.1"
   },
   "scripts": {
+    "postinstall": "webpack -p --config webpack/production.config.js",
```
Comment (bardzusny) [inline package.json]: what about `"postinstall": "npm run build"`?

File: app/mailers/props_digest.rb [hunk 2]
```diff
@@ -2,6 +2,6 @@ class PropsDigest < ApplicationMailer
   def received(user, props)
     @props = props
     @user = user
-    mail(to: user.email, subject: 'Received props digest')
+    mail(to: user.email, subject: 'Received props digest') if AppConfig.sendgrid_password.present?
```
Comment (Szeliga) [inline app/mailers/props_digest.rb]: Add a .rubocop.yml file that will fix this stupid issue

File: lib/tasks/assets.rake [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  task :webpack do
+    sh "NODE_ENV=#{Rails.env} npm run build" # this runs a react_webpack_rails script
+  end
+
+  desc 'Remove compiled Webpack assets'
+  task :clobber_webpack do
+    rm_rf "#{Rails.application.config.root}/app/assets/javascripts/react_bundle.js"
+    rm_rf "#{Rails.application.config.root}/app/assets/stylesheets/react_bundle.css"
+  end
+end
```
Comment (Szeliga) [inline lib/tasks/assets.rake]: It would be good to have tests for this stuff

**Comment by Szeliga** [comment]
Add the rubocop.yml file in a different PR. But the tests for that rake task are needed

**Comment by mxaly** [comment]
:+1:

**Pull request merged**